devlib/target.py

```diff
@@ -1199,6 +1202,14 @@ def __init__(self, version_string):
             self.release = version_string
             self.version = ''
 
+        match = KVERSION_REGEX.match(version_string)
+        if match:
+            self.version_number = match.group('version')
+            self.major = match.group('major')
+            self.minor = match.group('minor')
+            self.sha1 = match.group('sha1')
+            self.rc = match.group('rc')
```

Comment:
Yes, please update -- you shouldn't assume member attributes will always be accessed via `getattr`.